README.md

```diff
@@ -2,39 +2,56 @@
 
 > An Elixir Authentication System for Plug-based Web Applications
 
-Ueberauth is an authentication framework that is heavily inspired by [Omniauth](https://github.com/intridea/omniauth).
-You could call it a port but it is significantly different in operation - but almost the same by concept. Huge hat tip to omniauth.
-
-It is a two-phase authentication framework that provides a clear API - allowing for many strategies to be created and shared within the community.
-
-Ueberauth provides only the initial authentication challenge - the initial OAuth flow, collecting the information from a login form, etc. It does not authenticate each request; that's up to your application. You could issue a token or put the result into a session for your applications needs. Libraries like [Guardian](https://github.com/hassox/guardian) can help you with that aspect of authentication. Ueberauth manages the initial challenge only.
-
-The two phases are `request` and `callback`. These phases are implemented by Strategies.
+Ueberauth is two-phase authentication framework that provides a clear API -
+allowing for many strategies to be created and shared within the community. It
+is heavily inspired by [Omniauth](https://github.com/intridea/omniauth). You
+could call it a port but it is significantly different in operation - but
+almost the same by concept. Huge hat tip to [Intridea](https://github.com/intridea).
+
+Ueberauth provides only the initial authentication challenge, (initial OAuth
+flow, collecting the information from a login form, etc). It does not
+authenticate each request, that's up to your application. You could issue a
+token or put the result into a session for your applications needs. Libraries
+like (Guardian)[https://github.com/hassox/guardian] can help you with that
+aspect of authentication.
+
+The two phases are `request` and `callback`. These phases are implemented by
+Strategies.
 
 ## Strategies
 
 Strategies are plugs that decorate or intercept requests (or both).
-Strategies implement the two phases and then may allow the request to flow through to your downstream plugs.
-Implementing the request and callback phases is optional, and if not implemented the request will be decorated with Ueberauth information
-and allowed to carry on through the pipeline.
-
-## Request Phase
 
-The request phase is where you request information about the user. This could be a redirect to an OAuth2 authorization URL
-or a form for collecting username and password. The request phase is concerned with
-only the collection of information. When a request comes in on the request phase URL the relevant strategy will receive the `request_phase!` call.
+Strategies implement the two phases and then may allow the request to flow
+through to your downstream plugs. Implementing the request and callback phases
+is optional, and if not implemented the request will be decorated Ueberauth
```

Comment:
This doesn't quite read right. Maybe something like:

Implementing the request and callback phases is optional depending on the strategies requirements. If a strategy does not redirect, the request will be decorated with Ueberauth information and allowed to carry on through the pipeline.